website/docs/docs/getting-started/01-Installation.md

```diff
@@ -23,7 +25,7 @@ Install the latest version:
 
 ```bash
 helm install kro oci://registry.k8s.io/kro/charts/kro \
-  --namespace kro \
+  --namespace kro-system \
```

Comment:
why do we want kro-system? i feel like this opinionation should be a separate PR, feel to big of a change here.

Reply:
let not make "drastic" changes that could be considered breaking for existing users. Let's make this PR about publishing static manifests and leave changing the default namespace discussion to a seperate issue/slack thread.
/hold

Reply:
i'm thinking from the perspective of leases namespace changing too, which could cause two controllers to be running in parallel. Maybe not that big of a deal.

Looks like everyone wants kro-system, let's do it :)

/unhold